tesseract_ros/tesseract_msgs/msg/EnvironmentCommand.msg

```diff
@@ -68,6 +70,11 @@ tesseract_msgs/SceneGraph scene_graph
 tesseract_msgs/Joint scene_graph_joint
 string scene_graph_prefix
 
-# CHANGE_JOINT_LIMITS Command
-string change_joint_limits_name
-tesseract_msgs/JointLimits change_joint_limits_joint_limits
+# CHANGE_JOINT_POSITION_LIMITS Command
+tesseract_msgs/StringLimitsPair[] change_joint_position_limits
+
+# CHANGE_JOINT_VELOCITY_LIMITS Command
+tesseract_msgs/StringDoublePair[] change_joint_velocity_limits
+
+# CHANGE_JOINT_ACCELERATION_LIMITS Command
+tesseract_msgs/StringDoublePair[] change_joint_acceleration_limits
```

Comment:
I was  bit confused by this at first. I thought a StringDoublePair was a string and a pair of doubles (which is what StringLimitsPair is), but I guess this is fine.

Reply:
Yea I was trying to figure out how best to model maps in ros messages since they are not directly supported. I ended up with:
- StringDoublePair = std::pair<string, double>
- StringLimitPair = std::pair<string, double[2]> 
- StringStringPair = std::pair<string, string>

Reply:
Well its not actually `std::pair<string, Type>` though is it? Its `key, double` or `key, std::pair<double>`

Like I said, its not a big deal. I guess I was just wondering if we wanted something like
- StringDouble = string, double
- StringDoublePair (or maybe even StringDoubleDoublePair) = string, std::pair<double, double>
- StringString = string, string


For what its worth, I think the ROS style guide discourages generic msgs. So StringLimitPair sort of fits the style guide

Reply:
> Well its not actually std::pair<string, Type> though is it? 

For maps and unordered the container is a key value pair.

```
 template<class _Key, class _Tp,
	   class _Hash = hash<_Key>,
	   class _Pred = std::equal_to<_Key>,
	   class _Alloc = std::allocator<std::pair<const _Key, _Tp> > >
    class unordered_map
```

> Like I said, its not a big deal. I guess I was just wondering if we wanted something like

I understand, just explaining where the naming convention came from, but open to other solutions for storing map data structures in ROS messages.